scripts/exglobal_land_analysis_prep.sh

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env bash
+
+################################################################################
+# exglobal_land_analysis_prep.sh
+#
+# This script processes the IMS input ascii/netcdf files onto the UFS model grid.
+# Inputs: IMS ascii/netcdf file, IMS index file (is model resolution specific) 
+#         and model restart files.
+# Outputs: file with i) IMS-derived snow cover fraction over land on UFS model 
+#         grid, and ii) snow depth derived form the IMS snow cover fraction, 
+#         using an inversion of the noah model snow depletion curve.
+#
+################################################################################
+#  Utilities
+RES=${CASE:1}
+
+#  Directories.
+export gdasapp_dir=${gdasapp_dir:-"${HOMEgfs}/sorc/gdas.cd/"}
+BKGDIR=${DATA}/bkg
+
+#  Filenames
+export IMS_IODA=${IMS_IODA:-"${HOMEgfs}/sorc/gdas.cd/build/bin/imsfv3_scf2ioda.py"}
+export CALCFIMS=${CALCFIMS:-"${HOMEgfs}/sorc/gdas.cd/build/bin/calcfIMS.exe"}
```

Comment:
Why are these not in `ush/` and `exec/` respectively?

Reply:
Also, they don't need to be exported

Reply:
The `imsfv3_sfc2ioda.py` and the `calcfIMS.exe` are from GDASApp. We might to link/copy them into `ush/` and `exec/`. Great suggestions, and I will check the `link_workflow.sh` for this. Thanks.

Reply:
Done this.